healthtools/scrapers/base_scraper.py

```diff
@@ -189,8 +156,15 @@ def make_soup(self, url):
         soup = BeautifulSoup(response.content, "html.parser")
         return soup
 
-    def format_for_cloudsearch(self, entry):
+    def format_doc(self, entry):
         '''
         Format entry into cloudsearch ready document
         '''
-        return {"id": entry["id"], "type": "add", "fields": entry}
+        return entry
+
+    def push_to_elasticsearch(self, result):
+        for i, record in enumerate(result):
+            es.index(index='healthindex', doc_type=type(self).__name__, id=i, body=record, timeout=1000)
```

Comment:
👍 for the use of a dynamic name.

Let's try and get the doc_type to be `doctors`, `nurses` etc.